steve-ui-thymeleaf/src/main/resources/static/css/style.css

```
@@ -0,0 +1,431 @@
+    position: relative;
+    width: 80%;
+    border: 1px solid #fe9393;
+    border-radius: 4px;
+    margin: 12px auto;
+    padding: 12px;
+}
+input:disabled { background-color:#dddddd; }
+input[type="submit"]:disabled, input[type="submit"]:hover:disabled, input[type="submit"]:active:disabled {
+	cursor: not-allowed;
+	background:-webkit-gradient(linear, left top, left bottom, from(#a5a5a5), to(#5b5b5b));
+	background:-moz-linear-gradient(top,  #a5a5a5,  #5b5b5b);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)